src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesViewModel.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+#nullable enable
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Windows.Input;
+using Microsoft.VisualStudio.PlatformUI;
+using Microsoft.VisualStudio.ProjectSystem;
+using Microsoft.VisualStudio.Services.Common;
+using NuGet.Configuration;
+using NuGet.ProjectManagement;
+using NuGet.VisualStudio;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    public class ConfigurationFilesViewModel
+    {
+        public ObservableCollection<string> ConfigurationFilesCollection { get; private set; }
+        public string? SelectedPath { get; set; }
+        public ICommand OpenConfigurationFile { get; }
+        private ISettings _settings;
+        private INuGetProjectContext _projectContext;
+
+        public ConfigurationFilesViewModel(ISettings settings, INuGetProjectContext projectContext)
+        {
+            ConfigurationFilesCollection = new ObservableCollection<string>();
+            OpenConfigurationFile = new DelegateCommand(ExecuteOpenConfigurationFile, IsSelectedPath, NuGetUIThreadHelper.JoinableTaskFactory);
+            _settings = settings;
+            _projectContext = projectContext;
+        }
+
+        private bool IsSelectedPath()
+        {
+            return SelectedPath != null;
+        }
+
+        private void ExecuteOpenConfigurationFile()
+        {
+            OpenConfigFile(SelectedPath);
+        }
```

Comment:
It would be better to only have 1 method that handles the command to open a config file. The binding should pass the SelectedPath as the CommandParameter and the parameter should be read in the ExecuteOpenConfigurationFile method. The way this is written works, but it introduces an extra method call.

Reply:
Do I still need to pass a command parameter if I always want to use whatever is in `SelectedPath` (i.e. the `SelectedItem` from `ListView`) to open the file?